packages/physics-physx/src/PhysXCharacterController.ts

```diff
@@ -81,7 +81,7 @@ export class PhysXCharacterController implements ICharacterController {
    * {@inheritDoc ICharacterController.setSlopeLimit }
    */
   setSlopeLimit(slopeLimit: number): void {
-    this._pxController?.setSlopeLimit(slopeLimit);
+    this._pxController?.setSlopeLimit(Math.cos((slopeLimit * Math.PI) / 180));
```

Comment:
Not an `angle` or a `radian`, but a `cos`?

Reply:
yes